gordon/router.py

```diff
@@ -120,6 +122,12 @@ def _get_next_phase(self, event_msg):
         return next_phase
 
     async def _route(self, event_msg, next_phase=None):
+        if not interfaces.IEventMessage.providedBy(event_msg):
+            msg = (f'Ignoring message "{event_msg}". Does not correctly '
+                   'implement `IEventInterface`.')
```

Comment:
I think you mean `IEventMessage`

Reply:
ffffff you're right

Reply:
@descent3me fixed! I feel like a dummy 🤦‍♀️